CHANGELOG.md

```diff
@@ -169,6 +168,25 @@ _Infrastructure:_
 - Added automated tests for Android example project ([#813](https://github.com/cossacklabs/themis/pull/813)).
 - Added automated tests for desktop Java example project ([#816](https://github.com/cossacklabs/themis/pull/816)).
 
+## [0.13.10](https://github.com/cossacklabs/themis/releases/tag/0.13.10), May 26th 2021
+
+**Deprecation Notice for CocoaPods users:**
+  - `themis/themis-openssl` subspec based on GRKOpenSSLFramework is deprecated and will be removed in Themis version 0.14.
+  - `themis/themis-boringssl` subspec based on BoringSSL is deprecated and will be removed in Themis version 0.14.
+
+Please, switch to the default option in your Podfile: `pod 'themis'`
+
+**Hotfix for Apple platforms:**
+
+- `themis` for CocoaPods now uses XCFrameworks, supports Apple Silicon, and OpenSSL 1.1.1k ([#828](https://github.com/cossacklabs/themis/pull/828)).
+- Updated Carthage examples to use Themis XCFramework ([#823](https://github.com/cossacklabs/themis/pull/823)).
```

Comment:
I think this is a wrong PR link

Reply:
Hm... it looks correct to me. Why do you think it's wrong?

Reply:
oh sorry my fault, I mis-read "828" and "823". this is a correct link